misc/dbt-materialize/tests/adapter/test_deploy.py

```diff
@@ -712,9 +707,30 @@ def test_full_deploy_process(self, project):
         run_dbt(["run"])
 
         created_schema = project.created_schemas[0]
-
-        project_config = f"{{deployment: {{default: {{clusters: ['quickstart'], schemas: ['{created_schema}']}}}}}}"
-        project_config_deploy = f"{{deployment: {{default: {{clusters: ['quickstart'], schemas: ['{created_schema}']}}}}, deploy: True}}"
+        sinks_schema = f"{created_schema}_sinks_schema"
+
+        project_config = f"""
+        {{
+            deployment: {{
+                default: {{
+                    exclude_clusters: ['sinks_cluster'],
```

Comment:
Something I just realized while reviewing the docs: shouldn't we automatically exclude clusters that contain sinks _or_ at least verify if there are clusters with sinks and prompt users to add them to `exclude_clusters`?

Reply:
We already check that and prompt the users:

https://github.com/MaterializeInc/materialize/blob/b373db3c2220e389189c0e8cdd6cd28909aa74fc/misc/dbt-materialize/dbt/include/materialize/macros/deploy/deploy_init.sql#L31-L53

Reply:
Open to suggestions on the automatically excluding clusters and schemas. But this might be a bit hidden from the users and they might not realize we are actually doing it.

Reply:
Just updated the `deploy_get_objects` macro to filter out any clusters and schemas that contain sinks automatically.